src/themes/common.tsx

```diff
@@ -13,9 +13,24 @@ export const dimensions = {
   },
   form: {
     button: {
-      small: '30px',
-      normal: '40px',
-      large: '60px'
+      height:  {
+        xSmall: '20px',
+        small: '30px',
+        normal: '40px',
+        large: '60px',
+      },
+      padding: {
+        xSmall: '0 10px',
+        small: '0 20px',
+        normal: '0 20px',
+        large: '0 20px',
+      },
+      iconSize: {
+        xSmall: 12,
+        small: 14,
+        normal: 16,
+        large: 20
+      }
```

Comment:
I think it might be better to group things by size here so we have for example:
```
...
xsmall: {
  height: `20px`,
  padding: `0 10px`, 
  iconSize: `12`
}
```

I think that group will make this area easier to update with theming tweaks and has the advantage in styled components of being able to output the object which will convert to CSS rather than mapping each property. (although I'm not sure if it's smart enough to filter out `iconSize`).

Reply:
I have updated the group as requested.

About the theming, I don't think I understand that part. I'm not sure if it's a request or if you are thinking aloud for some update of the future. Please let me know.